bundles/org.openhab.binding.bluetooth.bluez/src/main/feature/feature.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.bluetooth.bluez-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Comment:
Is it correct to have all these feature files for the separate bluetooth projects? In the large feature file they were all grouped together because they depend on the general bluetooth project. Also the `tinyb` library is specific for bluez and if separated should be in this feature file I would expect if it would work anyway?

Reply:
I don't think they should have a feature-file at all. Or another one which depends on bluetooth. This is about the same situation like in mqtt.

Reply:
If you develop for one of those specific bluetooth bundles you still want to perform a feature verification from within the directory. (continuous integration should be able to just build this directory including checks, tests and feature verification.)

At the same time those sub-bundle features should not end up in the karaf feature repository for openHAB, as we don't want users to see those sub-bundles.

I see one solution in:
1. having a feature file with all sub-bundles in the main bluetooth (and mqtt) bundle,
2. a "private" feature file (excluded from merge) in each sub-bundle to perform feature verification.

WDYT?

Reply:
I moved the aggregated features to the footer of the file and excluded the bundle-wise features from the aggregation. Also added features for the child-bundles.